packages/components/src/horizontal-bar-list/stats-card.tsx

```diff
@@ -14,27 +14,58 @@ const StatsCard = ( {
 	footerAction,
 	isEmpty,
 	emptyMessage,
-	metricLabel,
 	heroElement,
+	splitHeader,
+	metricLabel,
+	mainItemLabel,
+	additionalHeaderColumns,
 }: StatsCardProps ) => {
 	const translate = useTranslate();
 
+	const titleNode = titleURL ? (
+		<a href={ `${ titleURL }` } className={ `${ BASE_CLASS_NAME }--header--title` }>
```

Comment:
Should we follow the Calypso CSS naming convention to name this `${ BASE_CLASS_NAME }__header-title`? And also for a few other places?

REF: 
- https://github.com/Automattic/wp-calypso/blob/trunk/docs/coding-guidelines/css.md
- https://getbem.com/naming/

Reply:
Oh, actually if the titleNode could be a block too. In that case, the name would be something like:

`${ BASE_CLASS_NAME }-header__title`